### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by eliotanke**

### CODE REVIEW

**Comment by eliotanke** [comment]
aaa

**Comment by eliotanke** [comment]
bbb

File: index.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ const github_slack_username_map = {
 
 const getCreateMessage = (url, title) => `@${team_name} CR please: <${url}|${title}>`;
 const getCommentMessage = (url, requester, commenter) => `Hey @${requester}! ${commenter} <${url}|commented> on your pull request`;
+const getApprovedMessage = (url, requester, approver) => `Hey @${requester}! ${approver} <${url}|approved> your pull request`;
```
Comment (eliotanke) [inline index.js:34]: aaa

**Review by eliotanke — COMMENTED** [review]
bbb

**Review by elio303 — APPROVED** [review]
ccc

**Review by elio303 — CHANGES_REQUESTED** [review]
eee

File: index.js [hunk 2]
```diff
@@ -109,60 +110,53 @@ const getSlackMessageId = async (github_url) => {
     return Item.SLACK_MESSAGE_ID.S;
 };
 
-// ---------- Event handling logic
+// ---------- Event handlers
 
-exports.handler = async (event) => {
-    console.log("event: " + JSON.stringify(event));
+const handleCreate = async (pull_request) => {
```
Comment (elio303) [inline index.js:115]: aaaaaaa

Comment (elio303) [reply index.js:115]: aaaa

**Pull request closed without merging**